packages/stores/src/account/index.ts

```diff
@@ -815,6 +815,86 @@ export class OsmosisAccountImpl {
     );
   }
 
+  /**
+   * Withdraw all or some liquidity from a position.
+   *
+   * @param positionId ID of the position to withdraw from.
+   * @param liquidityAmount L value of liquidity to withdraw, can be derived from liquidity of position.
+   * @param memo Memo of the transaction.
+   * @param onFulfill Callback to handle tx fullfillment given raw response.
+   * @returns
+   */
+  async sendWithdrawConcentratedLiquidityPositionMsg(
+    positionId: string,
+    liquidityAmount: Dec,
+    memo: string = "",
+    onFulfill?: (tx: any) => void
+  ) {
+    const aminoMsg = {
+      type: this._msgOpts.clWithdrawPosition.type,
+      value: {
+        position_id: positionId,
+        sender: this.base.bech32Address,
+        liquidity_amount: liquidityAmount.toString(),
```

Comment:
Is the reason you are converting just so you can get the number of decimal places below? If so, can we use the `dp()` function in Decimal.js? The reason is because in general, type converting Dec or Int to and from JS can cause rounding errors, whether or not you perform a calculation. Hence my hesitation.

Reply:
I'm not aware of the `dp()` function?

Reply:
I'm assuming we are using Decimal.js? If so, there is a built in function to tell you the number of decimal places: https://mikemcl.github.io/decimal.js/#dp

Reply:
No if you look at our import statement, we're using Dec type from @keplr-wallet/unit

Reply:
From looking at the yarn lock it appears decimal.js is just used by Jest, no prod dep